submissions/description/cross_site_scripting_xss/reflected/self/guidance.md

```diff
@@ -0,0 +1,6 @@
+<!--
```

Comment:
If we don't have guidance, can/should this template be removed?

Reply:
Thanks for the pickup, this has been removed and the recommendation tweaked with the correct `HTTP` code block added

Reply:
Thanks for the pickup, this has been removed and the recommendation tweaked with the correct `HTTP` code block added